src/main/scala/edu/berkeley/cs/amplab/mlmatrix/TSQR.scala

```diff
@@ -16,6 +16,11 @@ import org.apache.spark.SparkContext._
 
 class TSQR extends RowPartitionedSolver with Logging with Serializable {
 
+  /**
+   * Returns only the R factor of a QR decomposition of the input matrix. This
+   * operation is not reproducible since the R factors inside the TSQR tree can
```

Comment:
Similar to below, you can just say that `the value of R returned may not be reproducible across multiple invocations` (instead of the thing about ordering reduce functions)